src/pages/Accounts/Modals/SendFunds.tsx

```diff
@@ -71,11 +73,28 @@ type AskSendFundsModalProps = {
 }
 
 export const AskTransferFundsModal: FC<AskSendFundsModalProps> = ({ isVisible, onFinish, senderAddress, onClose }) => {
-  const { accounts } = useAccounts();
+  const { accounts, selectedAccount } = useAccounts();
   const [recipientAddress, setRecipientAddress] = useState<string | Account | undefined>();
   const [amount, setAmount] = useState<string>('');
-  const { chainData } = useApi();
-  const { kusamaFee } = useFee();
+  const { chainData, api } = useApi();
+  const [kusamaFee, setKusamaFee] = useState('0');
+  const [isFeeLoading, setIsFeeLoading] = useState(false);
+
+  const getKusamaFee = useCallback(() => {
+    setIsFeeLoading(true);
+    return debounce(() => {
+      if (!selectedAccount || !api?.market) return;
```

Comment:
Just FYI if somehow there no account or api - we will be stuck on loading. If it happens - stuck loader will be our least problem anyway.